### PR TITLE
Wrap northstar user migrations in try/catch

### LINKED ISSUE #4616: Guzzle fails locally
`GuzzleHttp\Exception\RequestException: cURL error 3: <url> malformed (see http://curl.haxx.se/libcurl/c/libcurl-errors.html) in GuzzleHttp\Handler\CurlFactory::createRejection() (line 162 of /var/www/dev.dosomething.org/html/profiles/dosomething/libraries/guzzle/src/Handler/CurlFactory.php).
The website encountered an unexpected error. Please try again later.`

### PR DESCRIPTION
**Pull request by angaither**
Fixes #4616

### CODE REVIEW

File: lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module [hunk 1]
```diff
@@ -52,10 +52,16 @@ function dosomething_northstar_register_user($form_state) {
   $ns_user = dosomething_northstar_build_ns_user($user, $form_state);
   // Send that to NS.
   $northstar = new Northstar();
-  $response = $northstar->updateUser($ns_user);
-  if ($response->getStatusCode() == '200' && module_exists('stathat')) {
-    stathat_send_ez_count('drupal - Northstar - user migrated - count', 1);
+  try {
+    $response = $northstar->updateUser($ns_user);
+    if ($response->getStatusCode() == '200' && module_exists('stathat')) {
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module]: This should be checking against a string?  If so, should use `===` ?

File: lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module [hunk 2]
```diff
@@ -66,9 +72,14 @@ function dosomething_northstar_update_user($form_state) {
   $ns_user = dosomething_northstar_build_ns_user($user, $form_state);
 
   $northstar = new Northstar();
-  $response = $northstar->updateUser($ns_user);
-  if ($response->getStatusCode() == '200' && module_exists('stathat')) {
-    stathat_send_ez_count('drupal - Northstar - user updated - count', 1);
+  try {
+    $response = $northstar->updateUser($ns_user);
+    if ($response->getStatusCode() == '200' && module_exists('stathat')) {
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module]: Same

**Comment by angaither** [comment]
@aaronschachter updated

**Comment by aaronschachter** [comment]
:+1:

**Pull request merged**